IDataBase.h

```diff
@@ -0,0 +1,117 @@
+#ifndef DB_INTERFACE
+#define	DB_INTERFACE
+#include <iostream>
+
+using namespace std;
```

Comment:
It's really bad practice to use `using namespace` in headers, the will leak all namespace items to global if header is included. Move this to your cpp file

Reply:
Fixed